tests/test_derivatives.py

```diff
@@ -160,8 +165,13 @@ def test_fd_space_staggered(self, space_order, stagger):
         This test compares the discrete finite-difference scheme against polynomials
         For a given order p, the finite difference scheme should
         be exact for polynomials of order p
-        :param derivative: name of the derivative to be tested
-        :param space_order: space order of the finite difference stencil
+
+        Parameters
+        ----------
+        derivative :
```

Comment:
can drop here and above, no need for super docstrings in tests

Reply:
I guess you mean like full deleting 

Parameters
-------------
.
.
.
.
.

Reply:
i meant deleting the added part, ie
```
Parameters
------------
.....
```
and also removing the part you deleted.

Reply:
to delete